unlock-app/src/hooks/useTransferFee.ts

```diff
@@ -29,3 +31,32 @@ export const useTransferFee = ({ lockAddress, network }: TransferFeeProps) => {
     getTransferFeeBasisPoints,
   }
 }
+
+export const useGetTransferFeeBasisPoints = ({
+  lockAddress,
+  network,
+}: TransferFeeProps) => {
+  const web3Service = useWeb3Service()
+  return useQuery(
+    ['transferFeeBasisPoints', lockAddress, network],
+    async () => {
+      const [transferFeeBasisPoints, lock] = await Promise.all([
+        web3Service.transferFeeBasisPoints(lockAddress, network),
+        web3Service.getLock(lockAddress, network),
+      ])
+
+      const transferFeePercentage = (transferFeeBasisPoints ?? 0) / 100
+      const unlimitedDuration =
+        lock?.expirationDuration === UNLIMITED_KEYS_DURATION
+
+      // unlimited memberships could not be made soul-bound
```

Comment:
I dont understand this? 
Unlimited means they don't expire. Why would they be non transferable?

Reply:
ref from this PR https://github.com/unlock-protocol/unlock/pull/11299 

I was added `unlimitedDuration` with `allowTransfer`

Reply:
That still does not make sense. The fact that they do not expire does not mean they can';t be transferred. It just means the transfer cannot have fees. 

Lock manager should be able to transfer a key, whether there are fees or not.